apps/backend/.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "tabWidth": 2
```

Comment:
I am overall okay to have a prettierrc but we need to have an agreement on linter/prettier config or we will always commit files with a lot of changes that are not the scope of our PRs
After agreement would be nice to have some dedicated cleanup PRs

Reply:
I totally agree with this. We already had this file in the project I just added explicit defaults. All of this can be handled by husky commit hooks (they are already installed and set up we're just not using them I think). The main idea is that there is a global `.prettierrc` file that is enforced by `prettier --write` on commit and everybody can have their own local `.prettierrc` in the `.vscode` directory. I personally can't read code properly if it only has `2` spaces so I have `4` in my local config.

Reply:
Yeah no probs at all with this it is a go for me, just a little reminder for us to come up with a complete ruleset for prettier.